src/main/java/se/nackademin/Personell.java

```diff
@@ -0,0 +1,49 @@
+package se.nackademin;
+
+class Personell {
+	private String name;
+	private int age;
+	private String title;
+	private int salary; // per annum
+	private int balance;
+	
+	public String getName() { return this.name; }
+	public int getAge() { return this.age; }
+	public String getTitle() { return this.title; }
+	public int getSalary() { return this.salary; }
+	public int getBalance() { return this.balance; }
+
+	public boolean sell(boolean isValid) {
+		if (isValid)
+			this.salary += 1;
+		return isValid;
+	}
+
+	public boolean feedAnimal(Lemur timon) {
```

Comment:
Should not take a Lemur, but an Animal, in order to use the benefit of the interface (e.g. support feeding any kind of animal.)